reference/replica-operations.md

```diff
@@ -2,27 +2,26 @@
+ Then it will attempt to restore the desired state \(replica count\) by creating a new replica, following its replica placement rules.  If it succeeds, the nexus will "rebuild" that new replica - performing a full copy of all data from a healthy replica `Online`, i.e. the source.  This process can proceed whilst the volume continues to process application I/Os although it will contend for disk throughput at both the source and destination disks.     
 
-If a nexus is restarted, i.e. the Mayastor pod hosting it restarts, with the assistance of the control plane it will 'recompos' itself; all of the previous member replicas will be re-attached to it, if possible, including any that were in the state `CHILD_FAULTED`.  If faulted replicas can be re-connected successfully, then the control plane will attempt to rebuild them directly, rather than seek replacements for them first.  This edge-case therefore does not result in the retirement of the affected replicas; they are simply reused.  If they are successfully re-attached but then continue to encounter I/O failures the rebuild will fail and will not be attempted again.
+If a nexus is restarted, i.e. the Mayastor pod hosting it restarts, with the assistance of the control plane it will 'recompose' itself; all of the previous healthy member replicas will be re-attached to it.  If previously faulted replicas are available to be re-connected (`Online`), then the control plane will attempt to reuse and rebuild them directly, rather than seek replacements for them first. This edge-case therefore does not result in the retirement of the affected replicas; they are simply reused.  If the rebuild fails then we follow the above process of removing a `Faulted` replica and adding a new one.
 
-Once provisioned, the replica count of a volume can be changed by editing its corresponding MayastorVolume \(MSV\) custom resource object.  The value of the `Replica Count` field may be either increased or decreased and the control plane will attempt to reconcile the desired and actual state, following the same replica placement rules as described herein.  If the replica count is reduced, faulted replicas will be selected for removal in preference to healthy ones.
+Once provisioned, the replica count of a volume can be changed using the kubectl-mayastor plugin `scale` subcommand.  The value of the `num_replicas` field may be either increased or decreased and the control plane will attempt to reconcile the desired and actual state, following the same replica placement rules as described herein.  If the replica count is reduced, faulted replicas will be selected for removal in preference to healthy ones.
```

Comment:
Should we call out the fact that we can only adjust the replica count by 1 at the moment?